test/url_bool_case_test.py

```diff
@@ -0,0 +1,79 @@
+import os
+import unittest
+
+import fastly
+import fastly.models
+
+
+SERVICE_ID = os.environ['FASTLY_SERVICE_ID']
+API_AUTH_TOKEN = os.environ['FASTLY_API_TOKEN']
+BACKEND_NAME = 'Google'
+BACKEND_ADDRESS = 'google.com'
+DOMAIN_NAME = 'test.fastly.redbee.dev'
```

Comment:
It would be better to have a generic domain referenced:

```suggestion
DOMAIN_NAME = 'test.example.com'
```

Reply:
It would seem the current test suite needs to be manually edited on a per-customer fork anyway, and so using a domain that doesn't resolve to Fastly's servers (such as `test.example.com) shouldn't be an issue because when running the tests the user will likely want to use their own Fastly service to run this type of _integration_ based test.

Reply:
Regarding this.. This test was actually added to the PR as a way for you to verify the bug, as that was requested from the support ticket.
As you point out, this test does not make much sense in here, should I remove it?

Reply:
I Updated this now, and also moved it into the setup function